src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -61,17 +61,17 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults([
             'translations' => [
                 'window.selectAll' => $this->trans('Select all', 'Admin.Actions'),
-                'window.settingsUpdated' => $this->trans('Settings udpated', 'Admin.Actions'),
-                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Actions'),
+                'window.settingsUpdated' => $this->trans('Settings updated', 'Admin.Global'),
+                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Catalog.Notification'),
                 'window.unselectAll' => $this->trans('Unselect all', 'Admin.Actions'),
                 'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
-                'window.cantDisableCover' => $this->trans('You cannot disable the cover image.', 'Admin.Actions'),
+                'window.cantDisableCover' => $this->trans('Choosing another image as the cover will automatically uncheck this box.', 'Admin.Actions'),
```

Comment:
```suggestion
                'window.cantDisableCover' => $this->trans('Using another image as cover will automatically uncheck this box.', 'Admin.Actions'),
```
I slightly modified this one so it is even more consistent with the rest of the wording. :) Thank you